Applications/Forward/test/testForward.cpp

```diff
@@ -249,3 +223,37 @@ void testGait2354WithController() {
     CHECK_STORAGE_AGAINST_STANDARD(results, *standard, rms_tols,
         __FILE__, __LINE__, "testGait2354WithController failed");
 }
+
+void testGait2354WithControllerGUI() {
+
+    // The following lines are the steps from ForwardToolModel.java that
+    // associates the a previous (orgiModel) model with the ForwardTool
+    // instead of the Tool loading the model specified in the setup
+    ForwardTool forward("subject01_Setup_Forward_Controller.xml");
+    Model origModel(forward.getModelFilename());
+
+    Model* model = new Model(origModel);
+    model->initSystem();
+    
+    const std::string resultsDir{ "ResultsCorrectionControllerGUI" };
+
+    forward.setResultsDir(resultsDir);
+    forward.updateModelForces(*model, "");
+    forward.setModel(*model);
+
+    model->initSystem();
+
+    forward.run();
+
+    // For good measure we'll make sure we still get the identical results
+    Storage results(resultsDir+"/subject01_states.sto");
+    //Storage standard("std_subject01_walk1_states.sto");
+    Storage standard("ResultsCorrectionController/subject01_states.sto");
+
+    int nstates = forward.getModel().getNumStateVariables();
+    int nq = forward.getModel().getNumCoordinates();
+    std::vector<double> rms_tols(2 * nstates, SimTK::SqrtEps);
+
```

Comment:
L241-244
```cpp
+    forward.updateModelForces(*model, "");
+    forward.setModel(*model);
+
+    model->initSystem();
```

Not sure it is worth adding another function for three lines.  I would also argue that if the `model.initSystem()` is required here, after setting the model on the Tool, then the logic/flow is is error prone. If you need access to the Tool's model, you should access it through the Tool's `getModel()` and get a reference to the same model. Otherwise you have two models, one in the GUI ForwardToolModel (Worker) and one within the `ForwardTool` itself and their state will be mismatched.

Reply:
@aseth1 While it may help the GUI stay consistent with tools, the point about making one function to use here is driven by repeated user requests to run tools on live models. Not critical to this PR tho.

Reply:
Maybe we can discuss this further. If it is primarily through the API that they want to run Tools on a live model, I don't think that is necessarily the best approach, but it should be possible if the GUI can do it.